apps/remix-ide/src/app/tabs/theme-module.js

```diff
@@ -77,7 +77,7 @@ export class ThemeModule extends Plugin {
     }
     const next = themeName || this.active   // Name
     const nextTheme = this.themes[next] // Theme
-    if (!this.forced) this._deps.config.set('settings/theme', next)
```

Comment:
we can't remove this test I think.
`this.forced` is set when the theme is set from the url parameter, in that case we don't want to persist that.
the issue could be that the value of `this.forced` is not correctly setup

Reply:
right. when we set it we compare null with undefined

Reply:
should be !!queryTheme instead